Mathlib/MeasureTheory/Integral/TorusIntegral.lean

```diff
@@ -99,7 +99,6 @@ def TorusIntegrable (f : ℂⁿ → E) (c : ℂⁿ) (R : ℝⁿ) : Prop :=
 
 namespace TorusIntegrable
 
--- Porting note (https://github.com/leanprover-community/mathlib4/issues/11215): TODO: restore notation; `neg`, `add` etc fail if I use notation here
```

Comment:
Can you comment on this note?

Reply:
I couldn't even work out what this was asking for. I'll just revert this file and ask the authors to investigate elsewhere.

Reply:
https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/notation.20in.20TorusIntegral/near/504502229